### PR TITLE
Aida 786

### PR DESCRIPTION
**Pull request by sharknc**
This ticket should be reviewed and merged back into AIDA-866 before AIDA-866 is merged to master. 

- Updated the batch-init script to force all submission validation result archives to be placed in an S3 directory instead of at the root s3 bucket. This is a requirement for the M18 evaluation. It is possible to place the validation results in a separate S3 bucket directory than the original submission itself. 
- Created batch-init-scheduler docker image with the init-scehduler.py script that will read in all submissions from a specific S3 directory and call the init.py script to initialize each as a separate job in AWS Batch
- Created README.md documentation for the batch-init and batch-init-scheduler docker containers in the batch-init directory
- Included information in README.md on how to create a cron job to run the batch-scheduler docker container at a particular time
- Updated all *.example scripts to read in the AWS credentials and region information from the AWS cli
- Moved all unused batch-multi node worker and main logic into batch-multi directory
- Removed batch-multi directory and all code within it. We will be moving forward with the batch-single approach and will no longer need this. 
- Removed the aida-validation-batch-cf-template.json CloudFormation template because batch-mutli is no longer needed
- Fixed aida-validation-batch-single-cf-template.json by removing invalid Default values for various parameters
- Updated the initialization script to generate SNS messages when a submission is invalid due to the structure (no NIST folder, no ttls in NIST folder, wrong extension, etc)

### CODE REVIEW

File: docker/batch-init/README.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+```bash
+crontab -e
+```
+
+Once the crontab editor has opened, add the following line and save. You will need to replace `<your-path>` with the the full path to the `run-scheduler.sh` script on your machine. By default, the log of the cron job will be placed in `/tmp/run-scheduler.log`. You can update this path to write the logs to any valid path on your machine. 
+
+Cron is driven by a time specification denoted by the five `*` at the beginning of the command. To specify a time when you would like your `run-scehduler.sh` script to be executed you will need to modify this command. More information on this specification can be found on the [Cron](https://en.wikipedia.org/wiki/Cron) wiki.
+
+```bash
+* * * * * <your-path>/docker/batch-init/run-scheduler.sh >> /tmp/run-scehduler.log 2>&1
```
Comment (jakevb) [inline docker/batch-init/README.md]: Typo: /tmp/run-scehduler.log should be /tmp/run-scheduler.log.

File: docker/batch-init/README.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+The Batch Scheduler Docker container can be set up to run via cron job at a specified time. In order to set up a cron job open cron tab with the following command:
+
+```bash
+crontab -e
+```
+
+Once the crontab editor has opened, add the following line and save. You will need to replace `<your-path>` with the the full path to the `run-scheduler.sh` script on your machine. By default, the log of the cron job will be placed in `/tmp/run-scheduler.log`. You can update this path to write the logs to any valid path on your machine. 
+
+Cron is driven by a time specification denoted by the five `*` at the beginning of the command. To specify a time when you would like your `run-scehduler.sh` script to be executed you will need to modify this command. More information on this specification can be found on the [Cron](https://en.wikipedia.org/wiki/Cron) wiki.
```
Comment (jakevb) [inline docker/batch-init/README.md]: Typo: run-scehduler.sh should be run-scheduler.sh.

File: docker/batch-init/README.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+Execute the run script with:
+
+```bash
+$ chmod +x run.sh
+$ ./run.sh
+```
+
+## Batch Scheduler
+
+Batch Scheduler Docker is an extension of the Batch Initializter which can be used for running Batch Initializations on more than one AIF Validation archive submission that is stored on S3. The only requirement for the Batch Scheduler is that all of the AIF submission archives must live in the same S3 object path. You do not need to build or deploy the Batch Initlaizer to use the Batch Scheduler. All Batch Initlizer logic will be included in the build Batch Scheduler Docker image.
```
Comment (jakevb) [inline docker/batch-init/README.md]: Typo: Initializter, Initlaizer, and Initlizer should be Initializer.

File: docker/batch-init/README.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+$ docker version -f '{{.Server.Experimental}}'
+```
+
+### Deploy Batch Single to AWS Batch
+
+The Batch Initializer assumes that the Batch Single Docker container has been deployed to ECR and the AWS Batch infrastructure has been appropriate deployed via CloudFormation. For more information on Batch Single, see the README.md in the `AIDA-Interchange-Format/docker/batch-single` directory. For more information on deploying the AWS Batch infrastructure with CloudFormation, see the README.md in `AIDA-Interchange-Format/infrastructure` directory. 
+
+## Batch Intializer
+
+The Batch Initializer Docker image is responsible for taking in a single AIF Validation archive submission, running validation on the submission contents and format, and submitting jobs to AWS Batch for validation. Also included in this repository is the Batch Scheduler Docker image. Batch Scheduler Docker is an extension of the Batch Initializter and is used for running Batch initializations on more than one AIF Validation archive submission that is stored on S3. See [Batch Scheduler](#Batch-Scheduler).
```
Comment (jakevb) [inline docker/batch-init/README.md]: Typo: Batch Initializter should be Batch Initializer.

File: docker/batch-init/README.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+Save, restart Docker via `service docker restart` and confirm experimental is enabled on the server. The following command will return `true` if experimental is enabled, `false` otherwise. 
+```
+$ docker version -f '{{.Server.Experimental}}'
+```
+
+### Deploy Batch Single to AWS Batch
+
+The Batch Initializer assumes that the Batch Single Docker container has been deployed to ECR and the AWS Batch infrastructure has been appropriate deployed via CloudFormation. For more information on Batch Single, see the README.md in the `AIDA-Interchange-Format/docker/batch-single` directory. For more information on deploying the AWS Batch infrastructure with CloudFormation, see the README.md in `AIDA-Interchange-Format/infrastructure` directory. 
+
+## Batch Intializer
```
Comment (jakevb) [inline docker/batch-init/README.md]: Typo: Intializer should be Initializer.

File: docker/batch-init/README.md (hunk 1 above)
Comment (sharknc) [reply docker/batch-init/README.md]: fixed

File: docker/batch-init/README.md (hunk 2 above)
Comment (sharknc) [reply docker/batch-init/README.md]: fixed

File: docker/batch-init/README.md (hunk 3 above)
Comment (sharknc) [reply docker/batch-init/README.md]: fixed

File: docker/batch-init/README.md (hunk 4 above)
Comment (sharknc) [reply docker/batch-init/README.md]: fixed

File: docker/batch-init/README.md (hunk 5 above)
Comment (sharknc) [reply docker/batch-init/README.md]: fixed

**Pull request merged**